resources/zib/zib-FunctionalOrMentalStatus.xml

```diff
@@ -44,6 +42,31 @@
         <comment value="FunctionalOrMentalStatus" />
       </mapping>
     </element>
+    <element id="Observation.extension:medicalDevice">
+      <path value="Observation.extension" />
+      <sliceName value="medicalDevice" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/workflow-supportingInfo" />
+      </type>
+      <isModifier value="false" />
+    </element>
+    <element id="Observation.extension:medicalDevice.value[x]">
+      <path value="Observation.extension.value[x]" />
+      <short value="MedicalDevice:MedicalDevice" />

```

Comment:
Should we just change this to 'MedicalDevice' and add a known issue?

Reply:
Yeah, we should. Wasn't there yet.

Reply:
Changed this and created a ticket